x-pack/plugins/task_manager/server/lib/log_health_metrics.ts

```diff
@@ -36,14 +36,15 @@ export function logHealthMetrics(
       capacity_estimation: undefined,
     },
   };
-  const statusWithoutCapacity = calculateHealthStatus(healthWithoutCapacity, config);
+  const statusWithoutCapacity = calculateHealthStatus(healthWithoutCapacity, config, logger);
   if (statusWithoutCapacity === HealthStatus.Warning) {
     logLevel = LogLevel.Warn;
   } else if (statusWithoutCapacity === HealthStatus.Error && !isEmpty(monitoredHealth.stats)) {
     logLevel = LogLevel.Error;
   }
 
   const message = `Latest Monitored Stats: ${JSON.stringify(monitoredHealth)}`;
+  const detectedProblemMessage = `Task Manager detected a degradation in performance. This is usually temporary, and Kibana can recover automatically. If the problem persists, check the docs for troubleshooting information: https://www.elastic.co/guide/en/kibana/current/task-manager-health-monitoring.html .`;
```

Comment:
We should probably generate the URL using the `docLinks` plugin. (example PR https://github.com/elastic/kibana/pull/92953/files).

Reply:
woops - I meant to leave a TODO for this (which would be easy to miss anyway), but there is no server-side docLinks plugin, yet - I did just add it as a TODO though :-)

> ### To Do
> open issue to change the new performance warning error to use server-side docLinks instead of the `current` reference, once we have server-side doclinks via this issue: [core] docLinks server-side service #95389 - the new issue will be blocked on that

Open for other ideas, felt like hard-coding `current` was going to be the best story for now ...

Reply:
Ahh that's true, it's only public.

What about something like the following?
```
import { kibanaPackageJson } from '@kbn/utils';

https://www.elastic.co/guide/en/kibana/${kibanaPackageJson.branch}/task-manager-health-monitoring.html
```

Reply:
I opened https://github.com/elastic/kibana/issues/109937 to track this (and deleted the TODO in the top comment).

Reply:
Ya, it does look like the existing docLink stuff makes use of the `branch` property, so seems doable.  Up to you - we'll need to thread it through all the calls ... :-)  I think long-term we still want docLinks for this, to handle cases when the URLs change (I assume it deals with that).

Reply:
Do we have a url shortening service? I vaguely remember using one in the past

Reply:
A URL shortener to make the doc link shorter?  I'm not sure I want that - I'd prefer the user see that we're pointing them to doc, vs pointing them to the great unknown :-).

Reply:
Turns out using `@kbn/utils` to get the patch version is absurdly easy: in commit https://github.com/elastic/kibana/pull/109741/commits/e8c9b0849dbb3293b9912febed9a73786feff2f6

Thanks for the tip, Mike!  I want to use more packages like this :-)